code/modules/gear_presets/survivors/trijent/preset_trijent.dm

```diff
@@ -112,3 +112,39 @@
 	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/lockable(new_human), WEAR_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/laceup(new_human), WEAR_FEET)
 	..()
+
+/datum/equipment_preset/survivor/security/trijent/cosurvivor
+	name = "CO Survivor - Trijent Dam - INCOM Officer"
+	languages = list(LANGUAGE_ENGLISH, LANGUAGE_JAPANESE)
+	job_title = JOB_TWE_RMC_CAPTAIN
+	role_comm_title = "IASF CPT"
+	assignment = "IASF CPT"
+	paygrades = list(PAY_SHORT_RNO2 = JOB_PLAYTIME_TIER_0)
+	faction_group = list(FACTION_TWE, FACTION_MARINE, FACTION_SURVIVOR)
+	minimap_icon = "rmc_commander"
+	idtype = /obj/item/card/id/dogtag
+	skills = /datum/skills/rmc/incom
+
+/datum/equipment_preset/survivor/security/trijent/cosurvivor/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/beret/royal_marine/team_leader, WEAR_HEAD)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/veteran/royal_marine/lt, WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine/veteran/royal_marine, WEAR_HANDS)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/royal_marine/knife, WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health, WEAR_EYES)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/accessory/patch/royal_marines, WEAR_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/accessory/storage/droppouch, WEAR_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/ammo_magazine/rifle/rmc_f90, WEAR_IN_ACCESSORY)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/gun/l905/full, WEAR_WAIST)
```

Comment:
full vp78 belt is pretty significant for survivors, not necessarily warranting a removal but something to keep in mind.